go_modules/spec/dependabot/go_modules/file_updater_spec.rb

```diff
@@ -104,9 +104,14 @@ module declares its path as: go.etcd.io/bbolt
       before do
         exit_status = double(success?: false)
         allow(Open3).to receive(:capture3).and_call_original
-        allow(Open3).to receive(:capture3).with(anything, "go get -d").and_return(["", stderr, exit_status])
+        allow(Open3).to receive(:capture3).with(anything, "go get -d github.com/etcd-io/bbolt@v1.3.5").and_return(["", stderr, exit_status])
       end
 
+      # This is failing and I'm not sure why...
+      # See also discussion here where it seems the author/reviewers agreed on one solution
+      # but then (confusingly) merged a different one w/o explaining why:
+      # https://github.com/dependabot/dependabot-core/pull/3482#discussion_r612343046
+      # So I'm unclear if this needs the regex, fixture, test, or something else to be updatd.
```

Comment:
Help?

I'm not really sure what needs to happen here...

Reply:
I think this is the same problem as https://github.com/dependabot/dependabot-core/pull/3590/files#r632456405